tests/system/python/packages/test_multiple_assets.py

```diff
@@ -253,45 +265,45 @@ def test_add_multiple_assets_before_after_restart(self, reset_fledge, start_nort
                 on endpoint GET /fledge/asset"""
 
         total_benchmark_services = 3
+        num_assets_per_service = (num_assets//(total_benchmark_services*2))
         # Total number of assets that would be created
-        total_assets = PER_BENCHMARK_ASSET_COUNT * total_benchmark_services
+        total_assets = num_assets_per_service * total_benchmark_services
```

Comment:
cannot we use num assets ?

Reply:
No, we cannot use num_assets here because if we use num_assets and num_assets contain even value then number of assets per services will not be equally divided, and test will fail. Since we are creating 3 or 6 services in different tests.